pkg/engine/engine.go

```diff
@@ -149,13 +150,28 @@ func Start(ctx context.Context, options ...EngineOption) *Engine {
 	})
 	e.prefilter = builder.Build(keywords)
 
-	ctx.Logger().V(2).Info("loaded decoders", "count", len(e.decoders))
-	ctx.Logger().V(2).Info("loaded detectors",
+	ctx.Logger().Info("loaded decoders", "count", len(e.decoders))
+	ctx.Logger().Info("loaded detectors",
 		"total", len(e.detectors[true])+len(e.detectors[false]),
 		"verification_enabled", len(e.detectors[true]),
 		"verification_disabled", len(e.detectors[false]),
 	)
 
+	// Sanity check detectors for duplicate configuration. Only log in case
+	// a detector has been configure in a way that we isn't represented by
```

Comment:
nit: a couple of typos

Reply:
also, for my own education, what does "configured in a way that isn't represented by the type and version" mean?

Reply:
Right now we're assuming the identity of a detector is uniquely captured by its type and version, but it's possible for a detector to be configured in a way such that the type and version are the same, but it scans in two different ways.